tests/unit/drivers/test_helper.py

```diff
@@ -74,39 +77,3 @@ def test_extract_docs():
     contents, docs_pts, bad_docs = DocumentSet([d]).all_embeddings
     assert len(bad_docs) == 0
     np.testing.assert_equal(contents[0], vec)
-
-
-def test_docgroundtruth_pair():
```

Comment:
why has this been removed?

Reply:
@JoanFM this part has been well tested in [primitive types test](https://github.com/jina-ai/jina/blob/master/tests/unit/types/document/test_helper.py)